adls-plugins/src/main/java/co/cask/hydrator/plugin/sink/ADLSBatchSink.java

```diff
@@ -128,10 +145,19 @@ public void transform(StructuredRecord input, Emitter<KeyValue<AvroKey<GenericRe
     @Macro
     public String fileSystemProperties;
 
-    @Description("Output schema for the output files.")
+    @Nullable
+    @Description("Output schema for avro format output files.")
     @Macro
     public String schema;
 
+    @Description("The format of output files.")
+    @Macro
+    private String outputFormat;
+
+    @Nullable
+    @Description("Field delimiter for text format output files.")
+    public String textDelimiter;
```

Comment:
name it `fieldDelimiter` instead of `textDelimiter`?

Reply:
maybe I'll rename it to `textFieldDelimiter` to stress that it's only needed for text format

Reply:
I agree with @anwar6953 about fieldDelimiter being a better name. textFieldDelimiter sounds like it's for text fields. I don't think the name here needs to mention the text format, but you can make the label in the widget json be more descriptive, like 'Field Delimiter (text format only)'.